locals.tf

```diff
@@ -89,4 +89,13 @@ locals {
   ti_src_dir  = "/opt/minio/ucfs-business-data-tarballs/"
   ti_wait     = "540"
   ti_interval = "1"
+
+
+  ti_shutdown = {
+    development = true
```

Comment:
this wouldn't actually work, Python has `True` and `False`. So like I mentioned in the other PR the type of this needs to be `bool` otherwise it will always shutdown the ASG if a value is present.